header.md

```diff
@@ -0,0 +1,3 @@
+<script>
+    console.log('test script from header.md');
```

Comment:
Do you still need this?

Reply:
No. As I mentioned in the description, these will be replaced with the FEDS implementation. I left it in anyway, since it's a feature branch 😃